conf/modules/fdm_crrcsim.xml

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="fdm_crrcsim" dir="fdm">
+  <doc>
+    <description>
+      CRRCSIM FDM backend for NPS simulator
+
+      Only for fixed-wing.
+      CRRCSIM is running as an external program using the MNAV inputdev.
```

Comment:
Can you add some extra information on what CRRCSIM is and how to start that.
Link to a wiki page would also suffice.